src/adapter/sources.ts

```diff
@@ -598,9 +633,18 @@ export class SourceContainer {
           undefined,
           undefined,
         );
+        await this._scriptSkipper.updateSkippingValueForScript(source);
```

Comment:
I'm not sure that I like calling this synchronously in source unwrapping. Verifying the absolute path involves reading and hashing the whole file, which may slow things down significantly when debugging large projects; in 951c8f68c0affe3f6c53e4cf4a77fc77ce327455 we saw a huge performance gain by moving hashing to a new thread and this change makes it somewhat synchronous again.

Reply:
I think this might be taken care of with the debouncer? If you could take another look